### PR TITLE
Add network-type to subnet

### LINKED ISSUE #32: Support for subnet type (ipv4 vs ipv6)
I can only add ipv4-subnet now with the importer.

### PR DESCRIPTION
**Pull request by andreabettich**
The import and dump supports the field network-type which can be IPv4 or IPv6.
This will fix #32

### CODE REVIEW

**Comment by jovandeginste** [comment]
Long time ago, I guess this is ok :-)

**Comment by andreabettich** [comment]
Yes, sorry about that. I hope the time of a respond and fix/implement will be a litte faster from now on.

File: config/example.yml [hunk 1]
```diff
@@ -121,6 +121,7 @@ foreman:
       tftp-proxy: Smart Proxy
       dns-proxy:
       boot-mode: DHCP
+      network_type: IPv4
```
Comment (karras) [inline config/example.yml]: Please `s/_/-/g` :)

**Review by karras — CHANGES_REQUESTED** [review]
Just one small remark regarding replacing the "_" with "-".

File: config/example.yml (hunk 1 above)
Comment (andreabettich) [reply config/example.yml]: Changed it. Thanks :)

**Comment by andreabettich** [comment]
@karras can we merge this?

**Review by karras — APPROVED** [review]

**Pull request merged**